data/campaigns/Eastern_Invasion/scenarios/10_Dark_Sanctuary.cfg

```diff
@@ -0,0 +1,2776 @@
+        [message]
+            speaker=unit
+            message= _ "Hmm, looks like a magic lamp. I’ve heard stories that these can grant wishes. Today must be my lucky day!"
+        [/message]
+
+        {QUAKE cave-in.ogg}
+        [delay]
+            time=1500
+        [/delay]
+
+        [sound]
+            name=melee-fire.ogg
+        [/sound]
+
+        # the petrification traps are pretty good against this guy, if you can lure him onto them
+        {NAMED_UNIT 2 (Elder Jinn) 42 11 "Jinn" "" ()} {FACING se} {ANIMATE}
+        [object]
+            [filter]
+                type=Elder Jinn
+            [/filter]
+
+            [effect]
+                apply_to=hitpoints
+                heal_full,increase_total=yes,{ON_DIFFICULTY -45 -15 0}
+            [/effect]
+
+            [effect]
+                apply_to=alignment
+                set={ON_DIFFICULTY lawful neutral chaotic}
+            [/effect]
+        [/object]
+
+        [delay]
+            time=500
+        [/delay]
+
+        [message]
+            speaker=Jinn
+            message= _ "WHO DARES DISTURB MY SLUMBER?"
+        [/message]
+    [/event]
+
+    # worthless green cloak
+    [event]
+        name=moveto
+        [filter]
+            side,x,y=1,38,10
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "This fine silken cloak would be priceless back in Wesnoth, but is useless to us here."
+        [/message]
+    [/event]
+
+    # open vault chests
+    [event]
+        name=moveto
+
+        [filter]
+            side,x,y=1,43,10
+        [/filter]
+
+        {REMOVE_IMAGE 43 10}
+        {PLACE_IMAGE "items/chest-open.png~FL(horiz)" 43 10}
+        {VARIABLE gold 45}
+        [message]
+            speaker,sound=unit,gold.ogg
+            message= _ "This chest contains $gold gold!"
+        [/message]
+        [gold]
+            side,amount=1,$gold
+        [/gold]
+    [/event]
+    [event]
+        name=moveto
+
+        [filter]
+            side,x,y=1,42,9
+        [/filter]
+
+        {REMOVE_IMAGE 42 9}
+        {PLACE_IMAGE "items/chest-open.png~FL(horiz)" 42 9}
+        [message]
+            speaker=unit
+            message= _ "It’s full of... toilet paper?!?"
+        [/message]
+    [/event]
+
+    # unlock treasure room
+    [event]
+        name=moveto
+        [filter]
+            side=1
+            x=39,40
+            y=11,11
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "I can unlock this gate from the inside."
+        [/message]
+        {MODIFY_TERRAIN Ias^Pr\o 39 12}
+    [/event]
+
+    #---------------------------
+    # EAST HALL, NORTH EVENTS
+    #---------------------------
+    # study guard
+    [event]
+        name=enter_hex
+        [filter]
+            side,x,y=1,29,7
+        [/filter]
+
+        [cancel_action]
+        [/cancel_action]
+        [message]
+            speaker=Study Guard
+            message= _ "Who. Comes."
```

Comment:
Missing question mark?

Reply:
I think, judging by the character's next line, it's just meant to reflect a certain style of speech

Reply:
I got the text style from that context, but a question mark would still be warranted. Unless it's to convey some sort of monotone voice, I suppose.

Reply:
yeah, conveying a monotone voice was how I read it